### DISCUSSION TITLE
Does React's `useSyncExternalStore` have any use for this library?

### DISCUSSION THREAD

**Discussion by jacobtipp** [crimx/observable-hooks#126] [Ideas]
I am wondering if there is any usecase for `useSyncExternalStore` with this library. It's supposed to help with concurrent rendering and most modern state management libraries now use this in favor of a `useEffect`. 

Just curious what are your thoughts on this?

**Comment by crimx** [comment]
Like you said "state management", `useSyncExternalStore` requires a way to access current value. It expects a "store", something like the Svlete Store, Signal, or `BehaviorSubject`. RxJS is more than a state management library. It is pull based fundamentally so `useSyncExternalStore` does not work with pure observable.
